README.md

```diff
@@ -53,9 +53,50 @@ There are three steps to the CI workflow:
      doesn't update any services, only manifests, so the push from the `deploy`
      job will never run another `deploy` job.
 
-## WIP: Deploy Slack and Hr service locally
+## Running on local Dapr Instance
 
-Use the 'docker-desktop' local k8s cluster.
+Make sure Docker is running (kubernetes is not required)
+
+Start Dapr locally
+
+```
+dapr init
+```
+
+Run the Slack and Hr Apis
+
+```
+cd lib/slack
+dapr run --app-id slack --app-port 3001 npm start
```

Comment:
I find it sad beyond words that dapr doesn't expose the `--app-port` argument as an environment variable to the child process.

Reply:
I suppose it would make the transition to kubernetes harder though? :/

Reply:
I agree, it's a shame that you have to manage all the port numbers when running locally, and basically in the source code of each api as opposed to in one central 'dapr configuration' type place.

I don't think it would make much difference to kubernetes, as each api runs in its own container so there are no clashing port number issues. I could be missing your point though ....